samples/client/petstore/elixir/lib/openapi_petstore/api/fake.ex

```diff
@@ -117,7 +117,7 @@ defmodule OpenapiPetstore.Api.Fake do
   - `{:ok, boolean()}` on success
   - `{:error, Tesla.Env.t}` on failure
   """
-  @spec fake_outer_boolean_serialize(Tesla.Env.client, keyword()) :: {:ok, boolean()} | {:error, Tesla.Env.t}
+  @spec fake_outer_boolean_serialize(Tesla.Env.client, keyword()) :: {:ok, Boolean} | {:error, Tesla.Env.t}
```

Comment:
`Boolean` should be `boolean()`

https://hexdocs.pm/elixir/1.15.0/typespecs.html#built-in-types